tcks/microprofile-opentelemetry/src/test/resources/exclusions.txt

```diff
@@ -0,0 +1,5 @@
+# Optional tests
+org.eclipse.microprofile.telemetry.tracing.tck.rest.JaegerPropagationTest
+org.eclipse.microprofile.telemetry.tracing.tck.rest.B3MultiPropagationTest
+org.eclipse.microprofile.telemetry.tracing.tck.rest.B3PropagationTest
+org.eclipse.microprofile.telemetry.tracing.tck.async.JaxRsServerAsyncTest
```

Comment:
Should we be excluding this one?

Reply:
Jaeger and B3 are marked as optional in the TCK. The JaxRsServerAsycn requires Jakarta Concurrency, which is not part of the MP Platform.